default-launch.json

```diff
@@ -2,7 +2,7 @@
     // Use IntelliSense to learn about possible attributes.
     // Hover to view descriptions of existing attributes.
     // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
-    "version": "0.2.0",
+    "version": "0.3.0",
```

Comment:
This version has nothing to do with the plugin runner version! The config here is for the VSCode Debugger.